lib/parallel_tests/grouper.rb

```diff
@@ -28,7 +28,39 @@ def in_even_groups_by_size(items, num_groups, options = {})
           raise 'Number of isolated processes must be less than total the number of processes'
         end
 
-        if isolate_count >= 1
+        if options[:specify_groups]
```

Comment:
lots of code here ... break it out into a method ?

Reply:
will do 👍